### PR TITLE
Release 4.6.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.6.0](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.6.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.6.0 (2021-02-25)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.6.0 (2021-02-26)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+- Added responsive container classes to the vertical tab component to allow child components to utilize correct responsive styles. [#154](https://github.com/blackbaud/skyux-tabs/pull/154)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added responsive container classes to the vertical tab component to allow child components to use the correct responsive styles. [#154](https://github.com/blackbaud/skyux-tabs/pull/154)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.6.0](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-release-4.6.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**